modules/ducktests/tests/ignitetest/services/utils/ignite_persistence.py

```diff
@@ -45,15 +47,17 @@ def init_persistent(self, node):
         """
         node.account.mkdirs(self.PERSISTENT_ROOT)
         node.account.mkdirs(self.TEMP_DIR)
+        node.account.mkdirs(self.LOGS_DIR)
```

Comment:
each ssh spent 300 ms on the test cluster.
could we pass all the dirs at one mkdirs()?